### DISCUSSION TITLE
contamination or polymorphism!

### DISCUSSION THREAD

**Discussion by jmhallas** [Kinggerm/GetOrganelle#303] [Q&A]
Hello,

My data is low coverage (<1x), and I have a number of assemblies that have the following error -

`WARNING: The graph might suffer from contamination or polymorphism!`

I am using the follow kmers [21,45,65,85,105].  I am wondering if there are other reasons I may get this error other than contmination?  I have removed these individuals from subsequent popgen analyses and would like to include them, but would like a little more information about this error.  

Thank you for your advise.

-josh

**Comment by JianjunJin** [comment]
It may help to troubleshoot the issue by seeing the complete log file and the assembly graph (*fastg or visualized)

**Reply by jmhallas** [reply]
Here is the log file from one individual.

[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/13781684/get_org.log.txt)

**Reply by JianjunJin** [reply]
Sorry for the unintentional slip due to other commitments.
Do you happen to still have the assembly graph for this run? Visualized graph is crucial for troubleshooting.
